### PR TITLE
SUPESC-240: Standalone testing (Old version)

### PR DESCRIPTION
**Pull request by luninaleksey**

### CODE REVIEW

File: phpstan.neon [hunk 1]
```diff
@@ -7,9 +7,15 @@ parameters:
     checkGenericClassInNonGenericObjectType: false
     checkMissingIterableValueType: false
     ignoreErrors:
-        - '#Expression on left side of ?? is not nullable\.#'
+        - '#Expression on left side of \?\? is not nullable\.#'
         - '#Parameter .+ of method .+ has invalid type .+#'
         - '#Method .+ has invalid return type .+#'
+        - '#Method .+ should return .+ but returns .+#'
```
Comment (dereuromark) [inline phpstan.neon:13]: Silencing those sounds dangerous, cant this be fixed inside code?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -1,124 +1,142 @@
...
+              run: composer validate
 
-            -   name: 'Run Elasticsearch'
-                uses: elastic/elastic-github-actions/elasticsearch@master
-                with:
-                    stack-version: 7.6.0
+            - name: Composer install
+              run: |
+                  composer --version
+                  composer install
```
Comment (dereuromark) [inline .github/workflows/ci.yml:85]: you could try composer update here.

File: composer.json [hunk 3]
```diff
@@ -37,7 +37,11 @@
         "spryker/propel": "*",
         "spryker/router": "*",
         "spryker/silex": "*",
-        "spryker/testify": "*"
+        "spryker/testify": "*",
```
Comment (dereuromark) [inline composer.json:40]: Best to put actual versions here, maybes prefer lowest much faster.
Also phpstan ^1.2.0 etc

File: .github/workflows/ci.yml (hunk 2 above)
Comment (luninaleksey) [reply .github/workflows/ci.yml]: Fixed.

File: composer.json (hunk 3 above)
Comment (luninaleksey) [reply composer.json]: Fixed.

**Comment by dereuromark** [comment]
Can you make a new branch please?
The failure to add proper gitignore is most likely a critical failure now, and requires a fresh/different branch name for cache busting.

**Comment by dereuromark** [comment]
Please also note:
With https://github.com/spryker/kernel/releases/tag/3.61.1 no more phpstan silencing needed for the IDE things

**Pull request merged**